accounting_app/accounting_app/doctype/purchase_invoice/purchase_invoice.json

```diff
@@ -16,7 +16,7 @@
   "total_qty",
   "total_amount",
   "credit_to",
-  "expense_account",
+  "debit_to",
```

Comment:
I prefer old name bacuase if you cancel this invoice debit convert to credit so this is a confusing name